doc/conf.py

```diff
@@ -34,10 +38,17 @@
 
 # Auto-generate header anchors with MyST parser
 myst_heading_anchors = 4
+# reference: https://myst-parser.readthedocs.io/en/latest/syntax/optional.html
 myst_enable_extensions = [
     "attrs_inline",  # Allow inline attributes after images
     "colon_fence",  # Allow code fences using colons
+    "substitution",  # Allow substituitions
 ]
+myst_substitutions = {
```

Comment:
```suggestion
# These enable substitutions using {variable} in the rst files
myst_substitutions = {
```

Reply:
`myst_substitutions`'s syntax is `{{ key }}` and is only available in myst-markdown files. I've updated the suggestion.